packages/client/src/runtime/core/init/resolveDatasourceUrl.ts

```diff
@@ -29,6 +29,16 @@ export function resolveDatasourceUrl({
 
   // env var is set for use but url is undefined
   if (datasourceUrl?.fromEnvVar !== undefined && resolvedUrl === undefined) {
+    if (TARGET_ENGINE_TYPE === 'edge') {
+      throw new PrismaClientInitializationError(
+        `error: Environment variable not found: ${datasourceUrl.fromEnvVar}.
+In edge runtimes, only \`process.env\` & \`globalThis\` are read, not \`.env\`.
```

Comment:
Wouldn't it be more correct to say that Prisma Client in edge runtimes does not read and load the `.env` file?
So only values that are already in `process.env` and/or `globalThis` via other sources can be accessed?

Reply:
How would you change this sentence?